lib/ngComponents.js

```diff
@@ -0,0 +1 @@
+/*** AUTO-GENERATED BY GULP ***/
```

Comment:
Yes, I think that will be better. We also need to be able to work on this module itself.